android/app/src/fdroid/res/layout/fragment_map.xml

```diff
@@ -9,4 +9,11 @@
         tilesource="Cyclemap"
         android:layout_width="match_parent"
         android:layout_height="match_parent" />
+
+    <ZoomControls
+        android:id="@+id/zoomControls"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center|bottom"
+        android:paddingBottom="50dp"/>
```

Comment:
Why the fixed padding?
We should set it programmatically above the nav bar.
I understand that ideally the nav bar is 48dp, but this might not be the case with every device out there.
Many OEMs choose not to include it at all.
I'd suggest to add a fixed margin of 4dp around the `ZoomControls` and then add a bottom padding equal to the nav bar size.